akka-stream/src/main/scala/akka/stream/scaladsl/Restart.scala

```diff
@@ -464,7 +464,7 @@ private abstract class RestartWithBackoffLogic[S <: Shape](
         if (finishing || maxRestartsReached()) {
           fail(out, ex)
         } else {
-          log.error(ex, "Restarting graph due to failure")
+          log.warn(ex, "Restarting graph due to failure")
```

Comment:
I'm afraid it's `log.warning` ;)

Reply:
Darn, that's what I get for just letting your build system do the building.